Patch104pZH/GameFilesEdited/Data/INI/Voice.ini

```
@@ -5678,3 +5678,18 @@ AudioEvent EvaVoiceUpgradeFortifiedStructure
   Volume = 90
   Type = ui voice player
 End
+
+; Patch104p @tweak Stubbjax 23/09/2022 Added some new upgrade voice lines.
+AudioEvent EvaVoiceUpgradeSatelliteHackOne
+  Sounds = ecsat1up
+  Control = random
+  Volume = 90
+  Type = ui voice player
+End
+
+AudioEvent EvaVoiceUpgradeSatelliteHackTwo
+  Sounds = ecsat2up
+  Control = random
+  Volume = 90
+  Type = ui voice player
+End
```

```suggestion
End

````
Minor nitpick, all lines should end with a /n character,

---

Ah yeah, I did notice that but forgot about it.